components/Contact.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import Title from './Title'
+import HomeMadeButton from './shared/Button'
+import { Form, Col } from 'react-bootstrap'
+
+import styles from '../styles/Contact.module.css'
+
+const Contact = () => {
```

Comment:
in this form, where do I paste my rest API URL to send the data to?

Reply:
good question

Reply:
I don't know how forms work

Reply:
lol